README.md

```diff
@@ -1,45 +1,146 @@
-# litt
-Literature tool for searching all pdfs in a directory
+# Litt
+Literature tool for searching all PDFs in a directory.
 
-## Dependencies 
-The command-line tool `pdftotext` should be available on your system.
+### tl;dr
+**Installing**:
+```
+wget https://github.com/georgbuechner/litt/releases/download/v1.0.1/litt-ubuntu-latest 
+chmod +x litt-ubuntu-latest
+cp -f litt-ubuntu-latest /usr/local/bin/litt
+```
+(Replace version and platform accordingly)
+
+**Usage**:
+```
+litt <index-name> -i <path-to-documents>
+litt <index-name> <search-term>
+```
+
+![Title Image](images/title-smaller.gif)
```

Comment:
This is great, but is it possible to record this again, with a larger font size? I can barely see anything on my screen.

Or we could just increase the size of the image, possibly with html if md doesn't support this?